service/pixelated/authentication.py

```diff
@@ -27,37 +28,41 @@
 
 
 class Authenticator(object):
-    def __init__(self, leap_provider):
+    def __init__(self, leap_provider, recovery=False):
```

Comment:
I changed the variable to `recovery_session` and it is a boolean indeed.
Do you think it's clearer now, @anikarni ?